### DISCUSSION TITLE
Action for Swift Package stuck on macOS runner

### DISCUSSION THREAD

**Discussion by soumyamahunt** [community/community#38407] [Actions]
I am trying to create a release for my Swift package, but [the action for this is stuck and not progressing further even after a long time](https://github.com/SwiftyLab/ValidatableKit/actions/runs/3403558251/jobs/5660206527).

From the logs, I think the action is stuck at Swift Package dependency resolution, but when I run the same steps/command on my development machine I am not able to reproduce this issue. I will appreciate any help with this.

**Comment by ossanna16** [comment]
Hi there @soumyamahunt and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our official introduction thread
To connect with other new users, check out this Discussion to Meet Your GitHub Community Team, further your GitHub knowledge by working through our GitHub Skill 🆙 exercises, and participate in our weekly Fun Friday chats
